src/collective/cover/widgets/richtext.py

```diff
@@ -0,0 +1,50 @@
+import pkg_resources
+
+try:
```

Comment:
I think this is not necessary as this code will only be execute if plone.app.widgets is installed and you already are checking for that on `configure.zcml`.

also, for me is pretty important to document what you're doing here, so we can remove this in the future when we drop compatibility with Plone 4. that sais, please add docstrings at the module and class levels explaining why this is needed..